homeassistant/components/blebox/switch.py

```diff
@@ -0,0 +1,36 @@
+"""BleBox switch implementation."""
+from homeassistant.components.switch import DEVICE_CLASS_SWITCH, SwitchDevice
+
+from . import BleBoxEntity, create_blebox_entities
+from .const import DOMAIN, PRODUCT
+
+
+async def async_setup_entry(hass, config_entry, async_add):
+    """Set up a BleBox switch entity."""
+
+    product = hass.data[DOMAIN][config_entry.entry_id][PRODUCT]
+    create_blebox_entities(product, async_add, BleBoxSwitchEntity, "switches")
+    return True
+
+
+class BleBoxSwitchEntity(BleBoxEntity, SwitchDevice):
+    """Representation of a BleBox switch feature."""
+
+    @property
+    def device_class(self):
+        """Return the device class."""
+        types = {"relay": DEVICE_CLASS_SWITCH}
```

Comment:
Please make `types` a constant

Reply:
Fixed in https://github.com/home-assistant/core/pull/35371/commits/1dc2473971a977d9a1bb9f9af616efaf424c13c3

I somehow missed this, so thanks for reminding me!